packages/@o3r/components/testing/setup-jest.ts

```diff
@@ -1,2 +1,8 @@
 import 'jest-preset-angular/setup-jest';
 
+global.setImmediate ||= ((fn: any, ...args: any) => {
```

Comment:
I would not say it's needed, but there is no reason to override it if we have a native implementation
this function is supposed to available natively on Node but it is not when using Jest (presumably because of jsdom environment)
I don't want to override this method unless is does not exist